### PR TITLE
SAK-49963 Recent Announcements: Incorrect attribution of highlighting

### PR DESCRIPTION
**Pull request by hornersa**
Jira: https://sakaiproject.atlassian.net/browse/SAK-49963

### CODE REVIEW

File: announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements.vm [hunk 1]
```diff
@@ -371,7 +371,7 @@
 					#set ($rowCount=0)
 					<ul class="synopticList">
 					#foreach ($ann_item in $showMessagesList)
-						#set ($isHightlighted = $ann_item.getProperties().getProperty("highlight"))
+						#set ($isHightlighted = (true == $ann_item.getProperties().getProperty("highlight")))
```
Comment (ottenhoff) [inline announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements.vm:374]: getProperty() returns a String?

so `true` should be quoted as a string?

if it works like this, it seems like Velocity just not being very strict?

Comment (hornersa) [reply announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements.vm:374]: I borrowed the logic that's elsewhere in the file. If you do a search for "(true == $", you'll see other lines that do not quote true. Should these be changed as well?

Comment (hornersa) [reply announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements.vm:374]: I'll force push a revision that places the 3 instances in this template where I've identified true should be a string.

**Review by ottenhoff — APPROVED** [review]
Thanks!

**Review by ern — APPROVED** [review]

**Pull request merged**